public_html/profiles/social/modules/social_features/social_core/templates/page-hero-data.html.twig

```diff
@@ -20,6 +20,15 @@
 #}
 <div class="cover">
 {% if hero_node or topic_type %}
+
+{% if node_edit_url %}
+  <div class="hero-action-button">
+    <a href="{{ node_edit_url }}" title="{#something that works for events and topics #}" class="waves-effect waves-light btn btn-raised btn-default btn-floating">
```

Comment:
Ja, inderdaad dat is de bedoeling. Die if conditie klopt ook nog niet, die had ik er als placeholder ingezet zodat een developer daar de juiste code in kan zetten. ;)